api_specs/v2/component_analyses.yaml

```diff
@@ -151,31 +151,31 @@ components:
         - recommendation:
             component_analyses:
               vulnerability:
-                - vendor_cve_ids: "CVE-2010-3082"
+                - cve_id: "CVE-2010-3082"
                   cvss: "1.2"
                   is_private: false
-                - vendor_cve_ids: "CVE-2018-1002"
+                - cve_id: "CVE-2018-1002"
                   cvss: "1.3"
                   is_private: false
             recommended_version : "2"
             message: "2 vulnerabilities within this dependency"
-            severity: "high"
+            highest_severity: "high"
             public_vulnerabilities_count: 2
             private_vulnerabilities_count: 1
             registeration_link: "https://app.snyk.io/login"
 
         - recommendation:
             component_analyses:
               vulnerability:
-                - vendor_cve_ids: "CVE-2018-1002"
```

Comment:
Ideally a vulnerability should have both id as well as cve_ids
id - Snyk id
cve_ids - list of cves associated with this vulnerability

Please refer https://github.com/fabric8-analytics/f8a-server-backbone/blob/master/src/v2/openapi.yaml#L145

Reply:
Do we really need Snyk Id..? Are we showing it to user..? Do we have any gains in putting it in response..?
In my opinion .. multiple IDs will not be required. It just creates a crowd/confusion for a user.  

Your thoughts..? @prashbnair

Reply:
Will it not be shown in the Clair integration. Snyk ID is useful as that is what will shown. It is what we show in SA also.

Reply:
Snyk id will be used in Clair integration, hope you remember the yesterday's demo in which devconsole vulnerability report included Snyk ids as well.

Reply:
@arajkumar As I mentioned earlier.. I will be incorporating all ur requested fields needed for ClairCore in the next iteration. For Now, We should be scoping on the present usecase only.

Reply:
@prashbnair Do we have any Use case of showing `Snyk Id`to user in the future, which I may not be aware of, in our current implementations..?

Reply:
Current V2 CA GET API is already exposing SYNK ID in vendor_cve_ids.

Sample response:
```
{
    "recommended_versions": "1.1.2",
    "registration_link": "https://app.snyk.io/login",
    "component_analyses": {
        "vulnerability": [
            {
                "vendor_cve_ids": "SNYK-PYTHON-FLASK-42185",
                "cvss": "7.5",
                "is_private": false
            }
        ]
    },
    "message": "flask - 0.12 has 1 known security vulnerability having high severity. Recommendation: use version 1.1.2.",
    "severity": "high",
    "known_security_vulnerability_count": 1,
    "security_advisory_count": 0
}
```

Reply:
@prashbnair What is your opinion on implementing https://issues.redhat.com/browse/APPAI-1411 as part of work? We would again need to spend time on API review if we do it in an separate PR.

Reply:
I leave it to @deepak1725 , I am fine either way.

Reply:
Thanks @prashbnair .
@arajkumar I assure you, I will take above-mentioned Task at Priority after this.